src/client/clientenvironment.h

```diff
@@ -141,6 +141,9 @@ class ClientEnvironment : public Environment
 	void updateCameraOffset(const v3s16 &camera_offset)
 	{ m_camera_offset = camera_offset; }
 	v3s16 getCameraOffset() const { return m_camera_offset; }
+
+	void updateFrameTime();
+	irr::u64 getFrameTime() { return m_frame_time; }
```

Comment:
Technically correct and yet inconsistent. Is there any reason to not use `u64` ?

Reply:
Indeed. Fixed.